### PR TITLE
adding categories

### PR DESCRIPTION
**Pull request by twissy22**

### CODE REVIEW

File: src/categories.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+  useEffect(() => {
+    getCategories().then((res) => {
+      setCategories(res.categories);
+    });
+  }, []);
+
+  return (
+    <ul className="ulcategories">
+        <li><a class="active" href="#home"><Link to="/" >Home</Link></a></li>
```
Comment (hevdevs) [inline src/categories.js]: Couple of errors here from nesting <a> tag (DOM attribute) and Links, highly preferable to just use the link instead, you can still apply class stylings using the className="" syntax!

File: src/reviews.js [hunk 2]
```diff
@@ -1,18 +1,20 @@
 import { useEffect, useState } from "react";
 import { getReviews, getReview } from "./api";
 import {Link} from 'react-router-dom'
+import { useParams } from "react-router-dom";
 
 const Reviews = () => {
   const [reviews, setReviews] = useState([]);
-
+  const { category } = useParams();
+  console.log(category)
```
Comment (hevdevs) [inline src/reviews.js]: remove console.log before merge

File: src/api.js [hunk 3]
```diff
@@ -3,8 +3,11 @@ const reviewsApi = axios.create({
   baseURL: "https://magnificent-hospital-gown-duck.cyclic.app/api",
 });
 
-export const getReviews = () => {
+export const getReviews = (category) => {
   let path = "/reviews";
+  if (category){
+    path+= `/?category=${category}`
+  }
   return reviewsApi.get(path).then((data) => {
```
Comment (hevdevs) [inline src/api.js]: For a neat refactor, look up how to use axios params

**Review by hevdevs — APPROVED** [review]
Hey Adam! functionality looks great! couple of suggestions here, happy for you to merge once cleaning up those link errors 😄

File: src/api.js (hunk 3 above)
Comment (twissy22) [reply src/api.js]: thanks, have done that much easier, i did try it but got confused the first time but understand it now

**Pull request merged**